functions/zoom-uploader.py

```diff
@@ -341,16 +341,10 @@ def add_tracks(self):
 
         logger.info("Adding tracks")
 
-        def sorted_videos(vids):
-            return sorted(
-                    vids,
-                    key=lambda vid: vid.metadata['track_sequence']
-                   )
-
         if self.speaker_videos is not None:
-            videos = sorted_videos(self.speaker_videos)
+            videos = self.speaker_videos
```

Comment:
Is this because we assume the order in which the tracks are added doesn't matter now?

Reply:
Yes. Getting rid of unnecessary code and metadata.